src/dispatchers/simple.coffee

```diff
@@ -3,11 +3,13 @@ Context = require("../context")
 
 module.exports = class SimpleDispatcher
 
-  constructor: (@service, @handlers) ->
-    for resource, actions of @service.default_handlers
-      handlers[resource] ||= {}
+  constructor: (@service, handlers) ->
+    @handlers = @service.default_handlers
+    
+    for resource, actions of handlers
+      @handlers[resource] ||= {}
       for name, handler of actions
-        handlers[resource][name] = handler
```

Comment:
That's a good point, I guess I just checked in what I had after I had been fiddling with it trying to figure out what was going on. I'll push an update to the simple version.